BUILD.bazel

```diff
@@ -31,6 +31,11 @@ cc_library(
     visibility = ["//:__subpackages__"],
 )
 
+# Protobuf runtime: this is the primary C++ public target.
```

Comment:
I'm confused, the comment is calling "arena" the main protobuf target?

Reply:
Ugh, bad merge. Fixed